overlay.d/05core/usr/lib/dracut/modules.d/35coreos-ignition/module-setup.sh

```diff
@@ -61,6 +61,7 @@ install() {
     install_ignition_unit "coreos-boot-edit.service" \
         "ignition-diskful.target"
 
+    install_ignition_unit coreos-ignition-unique-boot.service initrd.target
```

Comment:
We only want to do this on first boot, and only on machines with a boot device.

```suggestion
    install_ignition_unit coreos-ignition-unique-boot.service ignition-diskful.target
```